### PR TITLE
[FIX] remove duplicated view and fix missing field in elastic view

### PR DESCRIPTION
**Pull request by sebastienbeau**
This PR avoid duplicated code and solve the issue with missing tech_name and index_prefix_name 

image

### CODE REVIEW

**Comment by sebastienbeau** [comment]
@hparfr @PierrickBrun

File: connector_elasticsearch/views/se_backend_elasticsearch.xml [hunk 1]
```diff
@@ -1,61 +1,23 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-    <record model="ir.ui.view" id="se_backend_elasticsearch_form_view">
+    <record model="ir.ui.view" id="se_backend_algolia_form_view">
```
Comment (hparfr) [inline connector_elasticsearch/views/se_backend_elasticsearch.xml]: algolia form view in a elasticsearch.xml ?

File: connector_elasticsearch/views/se_backend_elasticsearch.xml [hunk 2]
```diff
@@ -1,61 +1,23 @@
...
-                        </field>
-                    </group>
-                </sheet>
+            <form position="attributes">
+                <attribute name="create">true</attribute>
+                <attribute name="edit">true</attribute>
             </form>
+            <field name="specific_backend" position="replace" />
+            <group name="se-main" position="inside">
+                <field name="es_server_host" placeholder="http://127.0.0.1:9200" />
```
Comment (hparfr) [inline connector_elasticsearch/views/se_backend_elasticsearch.xml]:
```suggestion
                <field name="es_server_host" placeholder="http://elastic:9200" />
```

**Review by hparfr — COMMENTED** [review]
Minor remarks, didn't test it yet

**Comment by sebastienbeau** [comment]
@hparfr fixed

**Pull request merged**